submission/submission-webapp/src/main/kotlin/ac/uk/ebi/biostd/data/service/SubmissionSqlDraftService.kt

```diff
@@ -0,0 +1,45 @@
+package ac.uk.ebi.biostd.data.service
+
+import ac.uk.ebi.biostd.persistence.common.request.PaginationFilter
+import ac.uk.ebi.biostd.persistence.common.service.SubmissionDraftService
+import ac.uk.ebi.biostd.persistence.model.DbUserData
+import ac.uk.ebi.biostd.submission.domain.service.SubmissionService
+import ebi.ac.uk.model.SubmissionDraft
+import org.springframework.transaction.annotation.Transactional
+import java.time.Instant
+
+open class SubmissionSqlDraftService(
```

Comment:
I'd suggest the name be `SubmissionDraftSqlService` since SubmissionDraft is the object in question and adding the "Sql" in the middle kind of breaks its meaning